core/src/test/java/org/opentosca/toscana/core/api/TransformationControllerTest.java

```diff
@@ -118,25 +152,45 @@ public void getTransformationProperties() throws Exception {
             .andExpect(jsonPath("$.properties[0].type").isString())
             .andExpect(jsonPath("$.properties[0].description").isString())
             .andExpect(jsonPath("$.properties[0].required").isBoolean())
+            .andExpect(jsonPath("$.properties[0].value").doesNotExist()) //Check that the value is null
             .andExpect(jsonPath("$.links[0].rel").value("self"))
             .andExpect(jsonPath("$.links[0].href")
                 .value("http://localhost/csars/k8s-cluster/transformations/p-a/properties"))
             .andReturn();
     }
 
     @Test
-    public void getTransformationPropertiesInvalidState() throws Exception {
+    public void getTransformationPropertyValues() throws Exception {
         preInitNonCreationTests();
-        ((DummyTransformation) csarService.getCsar("k8s-cluster").getTransformations().get("p-a"))
-            .setState(TransformationState.READY);
-        mvc.perform(
+        //Set a Property value
+        csarService.getCsar("k8s-cluster")
+            .getTransformations().get("p-a")
+            .getProperties().setPropertyValue("secret_property", "geheim");
```

Comment:
Please introduce variables for the strings, as they are used below for verification again.